mcs/class/System/System_xtest.dll.sources

```diff
@@ -0,0 +1,19 @@
+../../../external/corefx/src/CoreFx.Private.TestUtilities/src/System/AssertExtensions.cs
+../../../external/corefx/src/Common/tests/System/PlatformDetection.cs
+
+../../../external/corefx/src/Common/tests/System/Collections/ICollection.NonGeneric.Tests.cs
+../../../external/corefx/src/Common/tests/System/Collections/IEnumerable.NonGeneric.Tests.cs
+../../../external/corefx/src/Common/tests/System/Collections/IList.NonGeneric.Tests.cs
+../../../external/corefx/src/Common/tests/System/Collections/TestBase.NonGeneric.cs
+../../../external/corefx/src/common/tests/system/runtime/serialization/formatters/binaryformatterhelpers.cs
```

Comment:
the casing of the path looks wrong here (common vs Common) and it'll fail on Linux.

Reply:
oh yeah it's not the first tim I hit this thing - the list of files where extracted from corefx' crprojs 😞  thanks for noticing that, created this PR on windows

Reply:
interesting. this should fail in CoreFx too then I think? anyway, please send an upstream PR to align the csproj path casing with actual disk layout :)

Reply:
I guess MSBuild handles that ;-) but I'll create a PR in the corefx to cleanup such things